cartographer/mapping_2d/pose_graph.cc

```diff
@@ -555,6 +555,11 @@ sensor::MapByTime<sensor::OdometryData> PoseGraph::GetOdometryData() {
   return optimization_problem_.odometry_data();
 }
 
+sensor::MapByTime<sensor::FixedFramePoseData>
+PoseGraph::GetFixedFramePoseData() {
+  return {};  // Not implemented yet in 2D.
```

Comment:
Well, this is not ideal because it does not warn anybody, however, any warning would break the serialization function.
I guess 2D and 3D do behave differently, and this is a compromise so we can easily serialize both.

Reply:
Technically, it's not wrong - 2D _really has no FixedFramePoses_, so it returns an empty vector :-)